package.json

```diff
@@ -51,18 +52,18 @@
     "bump-version": "./script/bump-version.py",
     "check-tls": "python ./script/tls.py",
     "clang-format": "find atom/ brightray/ chromium_src/ -iname *.h -o -iname *.cc -o -iname *.mm | xargs clang-format -i",
-    "lint": "npm run lint:js && npm run lint:cpp && npm run lint:clang-format && npm run lint:py && npm run lint:docs",
-    "lint:js": "standard && cd spec && standard",
+    "lint": "./script/lint.js && npm run lint:clang-format && npm run lint:docs",
+    "lint:js": "./script/lint.js --js",
```

Comment:
I'm not 100% convinced by this, a common pattern at the moment to fix JS linting issues is `npm run lint:js -- --fix` because `standard` support `--fix` to automatically fix some simple issues like spacing and indenting.

With this change we lose all the power / flags that `standard` supports

Reply:
I suppose the question I'm asking is "why do we need these one liner CLI scripts in a more complicated abstraction script"

Reply:
I agree that standard is quite fast. JS wasn't my main motivation for this.

This got started because the python lint runner is broken in master, and I wanted to kill two birds with one stone by fixing it and rewriting it in JS. Some of the file-finding code looked a lot like what `cpplint.js` already had, so it seemed reasonable to throw it together. 

The python-specific parts of the script now take 10 lines, and the js-specific parts also take 10 lines, so I think this is a flexible approach without much downside.

Reply:
One nice bonus of doing this, the new linter found a bug in our old `pylint.py` code that didn't lint any of the files in `script/lib/`

Reply:
Can we make `--fix` work then if we're only running with `--js`.  I really like being able to do that